src/main/java/uk/gov/hmcts/reform/civil/service/PaymentsService.java

```diff
@@ -58,20 +90,15 @@ private CreditAccountPaymentRequest buildRequest(CaseData caseData) {
 
         String customerReference = ofNullable(generalAppPBADetails.getPaymentDetails())
             .map(PaymentDetails::getCustomerReference)
-            .orElse(generalAppPBADetails.getPbaReference());
```

Comment:
needs to be correct to always use pba reference

Reply:
Removed the mapping and mapping directly to the pbaRference